fcd_sale_order_rpt_invoice/report/fcd_sale_report_invoice_menuitem.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="action_fcd_sale_report_invoice" model="ir.actions.act_window">
+        <field name="name">Sales Margin per Operation</field>
+        <field name="res_model">fcd.sale.report.invoice</field>
+        <field name="view_mode">tree</field>
+        <field name="context">{"search_default_filter_date_order_this_month":1}</field>
+        <field name="help" type="html">
+          <p class="oe_view_nocontent_create">
+            Not possible.
+          </p><p>
+            Can't create or update records in this view.
+          </p>
+        </field>
+    </record>
+
+    <menuitem id="fcd_sale_report_invoice"
+        name="Sales Margin per Operation"
+        action="action_fcd_sale_report_invoice"
+        parent="sale.menu_sale_report"
+        groups="sales_team.group_sale_manager"
```

Comment:
As model already has limited red access for `group_sale_manager` , this is unnecessary IMO. Please verify anyway with other user role.